Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY ./examples/flex-debug.sh /opt/csi-provisioner/
 
 # install the go kube piece of provisioner
 
-COPY ./_output/csi-provisioner /opt/csi-provisioner/
+COPY ./bin/csi-provisioner /opt/csi-provisioner/
```

Comment:
let's make _output/bin for binaries. Change here and in Makefile and README too.

Reply:
It should be the same as attacher, which uses `bin/` too.

Reply:
I do not like bin/ for build artifacts.... It's a common directory for scripts or other real binaries that might be part of the project..  _output/bin would be fine.